cirq/protocols/json_test.py

```diff
@@ -667,6 +518,88 @@ def test_to_from_strings():
         cirq.read_json(io.StringIO(), json_text=x_json_text)
 
 
+def _eval_repr_data_file(path: pathlib.Path):
+    return eval(path.read_text(), {
```

Comment:
this seems mildly dangerous

Reply:
I agree in principle... but to be pragmatic we're _already_ parsing files and running them as python when we invoke pytest. I don't see any additional risk from this... if there was, it would be of the style "a tool was setup to detect people replacing code with delete everything commands, but the tool ignored things that didn't end with .py".

Reply:
yup, logically this isn't much different than the gaping security hole of python / pytest / everything already :)